samples/nrf9160/mqtt_simple/src/main.c

```diff
@@ -30,6 +30,24 @@ static bool connected;
 /* File descriptor */
 struct pollfd fds;
 
+#if defined(CONFIG_BSD_LIBRARY)
```

Comment:
I'm worried this might communicate to the user that it is supported to run this sample without BSD_LIBRARY. But that isn't supported, is it?

Reply:
Actually it is - currently you can run it with qemu_x86 (useful for development). But eventually, we should move it from `nrf9160` to a generic `net` subdirectory and be able to run it with any networking techonoly we support (bsdlib, OpenThread, IPSP etc.)

Reply:
Ah, okay, thanks